de-lib/src/main/java/org/iplantc/de/diskResource/client/presenters/DiskResourcePresenterImpl.java

```diff
@@ -644,14 +645,19 @@ public void go(HasOneWidget container,
                    final List<? extends HasId> diskResourcesToSelect) {
 
         if ((folderToSelect == null) || Strings.isNullOrEmpty(folderToSelect.getPath())) {
-            go(container);
+            go(container, true);
```

Comment:
Does this mean new data windows will open with the details panel collapsed by default?

It may not be obvious to new users that they can view more details of a selected item by clicking the `<<` button at the top-right of the grid (under the `Trash` menu and next to the folder name grid-header).

Reply:
This `go()` method is only used from folder selector. So I set them to true by default. But in general Data window opens with details panel expanded. Once you collapse the details panel, we store that as preference in the local storage.

Reply:
Ah, I didn't catch that this `go` method is only called from the file/folder select dialogs before.
And now I see where the collapsed settings is saved in the window (and not in the presenter) 👍